lib/shift.rb

```diff
@@ -25,7 +25,7 @@ def offset(date = nil)
 
   def final_offset_split(date = nil)
     if date == nil
-       @date_today**2
+       @date_today.to_i**2
```

Comment:
this is where I convert the date to integer, helps preserve the zeros in the begining